gr-blocks/include/gnuradio/blocks/control_loop.h

```diff
@@ -99,7 +107,13 @@ class BLOCKS_API control_loop
      * method in case another way is desired as this is fairly
      * heavy-handed.
      */
-    void phase_wrap();
+    void phase_wrap()
+    {
+        while (d_phase > M_TWOPI)
```

Comment:
I know that this is the traditional way to do phase wrap. I wonder if it would make any difference to wrap each phrase in an `if` ... for example:
```
if (d_phase) {
    while (d_phase > M_TWOPI)
    d_phase -= M_TWOPI;
} else {
    while (d_phase < -M_TWOPI)
         d_phase += M_TWOPI;
}
```
since they are totally separate: if the first `while` happens, then the 2nd won't & vice versa. It's a small issue, really. Just wondering.